### ISSUE TITLE
Marquee : Subcopy is gray instead of Black

### ISSUE THREAD

**Issue by georgediaz** [grommet/hpe-digitaltoolkit#78]
Please change subcopy to black from gray for accessibility purposes.

**Comment by georgediaz** [comment]
![screen shot 2016-06-15 at 8 59 16 am](https://cloud.githubusercontent.com/assets/445559/16095170/504eca06-32f8-11e6-8d26-49cd08bbf5e9.png)

**Comment by codeswan** [comment]
@kahaeda This has been implemented, and is available to view on the staging site.
